2022-01-11 15:22:23,977 INFO: Initialized curve data.
2022-01-11 15:22:23,983 INFO: Setting up initial twist data...
2022-01-11 15:22:25,005 INFO: Twist setup complete.
2022-01-11 15:22:25,005 INFO: Searching for rational points on each twist...
2022-01-11 15:22:26,337 INFO: Finished searching for rational points on each twist.
2022-01-11 15:22:26,337 INFO: Testing local solvability of twists...
2022-01-11 15:22:27,182 INFO: Finished local solvability testing.
2022-01-11 15:22:27,183 INFO: Computing elliptic curve a-invariants...
2022-01-11 15:22:32,814 INFO: Finished computation of a-invariants.
2022-01-11 15:22:32,815 INFO: Computing MW group (delta = ['20', '-49', '37', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1])...
2022-01-11 15:25:54,066 INFO: Finished MW computation (delta = ['20', '-49', '37', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1])
2022-01-11 15:25:54,068 INFO: Skipped MW computation because no base point (delta = ['400', '-960', '721', '-81', '-80', '20'])
2022-01-11 15:25:54,068 INFO: Computing MW group (delta = ['20', '-49', '36', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1])...
2022-01-11 15:27:48,989 INFO: Finished MW computation (delta = ['20', '-49', '36', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1])
2022-01-11 15:27:48,992 INFO: Computing MW group (delta = ['1'], g = [20, -48, 36, -4, -4, 1])...
2022-01-11 16:59:26,716 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 506, in <module>
    rank, MW_proven, orders, gens = twist_MW_group(D)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 329, in twist_MW_group
    A, mw, bool_rank, bool_index = magma.MordellWeilGroup(E, nvals=_sage_const_4 )
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1787, in __call__
    return M.function_call(self._name,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1162, in function_call
    return self._do_call(fun, nvals)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1221, in _do_call
    raise TypeError("Error executing Magma code:\n%s" % out)
TypeError: Error executing Magma code:

Current total memory usage: 16371.7MB, failed memory request: 32.0MB
System Error: User memory limit has been reached
2022-01-14 09:07:50,208 INFO: Loaded curve data from file.
2022-01-14 09:07:50,209 INFO: Reducing MW generators...
2022-01-14 09:07:50,678 INFO: Finished reducing MW generators.
2022-01-14 09:07:50,681 INFO: Running elliptic Chabauty (delta = ['20', '-49', '37', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1])...
2022-01-14 09:08:02,208 INFO: Chabauty completed successfully (delta = ['20', '-49', '37', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1]).
2022-01-14 09:08:02,298 INFO: Skipped Chabauty because no base point (delta = ['400', '-960', '721', '-81', '-80', '20'])
2022-01-14 09:08:02,299 INFO: Running elliptic Chabauty (delta = ['20', '-49', '36', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1])...
2022-01-14 09:08:13,758 INFO: Chabauty completed successfully (delta = ['20', '-49', '36', '-4', '-4', '1'], g = [20, -48, 36, -4, -4, 1]).
2022-01-14 09:08:13,761 INFO: Skipped Chabauty because MW generators haven't been computed. (delta = ['1'], g = [20, -48, 36, -4, -4, 1])
2022-01-14 09:08:13,884 INFO: Tasks complete. Recording data and exiting.
